python/py-pysam/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           python 1.0
 name                py-pysam
 set realname        pysam
 version             0.15.1
-revision            1
+revision            2
```

Comment:
There is no need to change the `revision` when adding sub-ports because nothing has changed for anyone who has one installed.